Description: This patch sets the dbconfig placeholders in config.inc.php.
Forwarded: not-needed
Author: Norman Messtorff <normes@normes.org>
Last-Update: 2012-05-28

Index: postfixadmin/config.inc.php
===================================================================
--- postfixadmin.orig/config.inc.php	2015-09-26 15:03:05.000000000 +0100
+++ postfixadmin/config.inc.php	2015-09-26 15:03:16.000000000 +0100
@@ -15,6 +15,11 @@
  * Contains configuration options.
  */
 
+// This loads the automatic generated DB credentials from /etc/postfixadmin/dbconfig.inc.php
+require_once('/etc/postfixadmin/dbconfig.inc.php');
+if (!isset($dbserver) || empty($dbserver))
+        $dbserver='localhost';
+
 /*****************************************************************
  *  !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 
  * You have to set $CONF['configured'] = true; before the
@@ -22,7 +27,7 @@
  * Doing this implies you have changed this file as required.
  * i.e. configuring database etc; specifying setup.php password etc.
  */
-$CONF['configured'] = false;
+$CONF['configured'] = true;
 
 // In order to setup Postfixadmin, you MUST specify a hashed password here.
 // To create the hash, visit setup.php in a browser and type a password into the field,
@@ -80,11 +85,11 @@
 // mysql = MySQL 3.23 and 4.0, 4.1 or 5
 // mysqli = MySQL 4.1+ or MariaDB
 // pgsql = PostgreSQL
-$CONF['database_type'] = 'mysqli';
-$CONF['database_host'] = 'localhost';
-$CONF['database_user'] = 'postfix';
-$CONF['database_password'] = 'postfixadmin';
-$CONF['database_name'] = 'postfix';
+$CONF['database_type'] = $dbtype;
+$CONF['database_host'] = $dbserver;
+$CONF['database_user'] = $dbuser;
+$CONF['database_password'] = $dbpass;
+$CONF['database_name'] = $dbname;
 // If you need to specify a different port for a MYSQL database connection, use e.g.
 //   $CONF['database_host'] = '172.30.33.66:3308';
 // If you need to specify a different port for POSTGRESQL database connection
